src/mockServer/Controllers/ProductController.ts

```diff
@@ -0,0 +1,80 @@
+import ProductService from "mockServer/Services/ProductService";
+import { RestRequest, RestContext, ResponseFunction } from "msw";
+import { IProduct } from "shared/models";
```

Comment:
The controller is sync regular function that returns a response as explained in MSW DOCS

Reply:
how is that a sync function if req.json to get body of request returns a promise?